apidump/apidump.go

```diff
@@ -245,19 +246,27 @@ func compileRegexps(filters []string, name string) ([]*regexp.Regexp, error) {
 func Run(args Args) error {
 	args.lint()
 
+	debugEnabled := viper.GetBool("debug")
+
+	if args.EnableOutbound {
+		printer.Warningln("Enabling collection of outbound traffic. The \"--enable-outbound-collection\" flag is deprecated and may be removed in a future release.")
+	}
```

Comment:
Can you add a debug statement here stating that we're opening extra packet captures to inspect filtered traffic?

I was confused at the interaction of debugEnabled and EnableOutbound, so it might be worth including the extra information in the output too.

Reply:
In 5e0a89a. I've also introduced `capturingOutbound := args.EnableOutbound || debugEnabled` with a comment at the top of the function to help clarify.